substrate/frame/support/src/traits/voting.rs

```diff
@@ -82,7 +82,7 @@ impl<T, P: Polling<T>> sp_runtime::traits::Get<u32> for ClassCountOf<P, T> {
 }
 
 pub trait Polling<Tally> {
-	type Index: Parameter + Member + Ord + PartialOrd + Copy + HasCompact + MaxEncodedLen;
+	type Index: Parameter + Member + Ord + PartialOrd + Copy + HasCompact + MaxEncodedLen + From<u8>;
```

Comment:
Also breaking change (major), and maybe also possibly to avoid it ba using the where bound on the benchmark.

Reply:
Good idea with the benchmark where bound.

Reply:
Maybe we don't even need it. Did not spot where this could be required

Reply:
> Also breaking change (major), and maybe also possibly to avoid it ba using the where bound on the benchmark.

The where bound could be

``` rust
#[instance_benchmarks(
where <<T as pallet::Config<I>>::Polls as frame_support::traits::Polling<Tally<T, I, pallet::Pallet<T, I>>>>::Index: From<u8>
)]
mod benchmarks {}
```
> Maybe we don't even need it. Did not spot where this could be required

required for 
```rust
<NoOpPoll as Polling<T>>::Index::MAX.into()
```